Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabOverrideLabelHandler.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabPropertyEditorNodes.h>
+#include <AzToolsFramework/UI/DocumentPropertyEditor/PropertyHandlerWidget.h>
+#include <AzQtComponents/Components/Widgets/ElidingLabel.h>
+
+#include <QHBoxLayout>
+#include <QIcon>
+#include <QMenu>
+#include <QToolButton>
+
+namespace AzToolsFramework::Prefab
+{
+    //! Class to handle the override label property when encountered in a DPE DOM.
+    //! Responsible for setting the ui/ux for overridden properties.
+    class PrefabOverrideLabelHandler
+        : public PropertyHandlerWidget<QWidget>
+    {
+    public:
+        inline static constexpr const char* OverriddenPropertyName = "overridden";
```

Comment:
I'm pretty sure the inline and the static don't do anything in this context, since the constexpr is just telling the compiler to do a replace during preprocessing, but I applaud the effort ;)